packages/chat-component/src/styles/chat-component.ts

```diff
@@ -46,12 +71,30 @@ export const chatStyle = css`
     margin-block-end: 0;
   }
   .button {
-    color: var(--text-color);
+    color: var(--c-text);
     border: 0;
     background: none;
     cursor: pointer;
     text-decoration: underline;
   }
+  @keyframes chatmessageanimation {
```

Comment:
this was moved out of here into their respective components

Reply:
Removed.